src/nvim/ex_cmds.lua

```diff
@@ -2678,18 +2678,18 @@ return {
     func='ex_ni',
   },
   {
-    command='tearoff',
-    flags=bit.bor(NEEDARG, EXTRA, TRLBAR, NOTRLCOM, CMDWIN),
-    addr_type=ADDR_LINES,
-    func='ex_tearoff',
-  },
-  {
     command='terminal',
     flags=bit.bor(BANG, FILES, CMDWIN),
     addr_type=ADDR_LINES,
     func='ex_terminal',
   },
   {
+    command='tearoff',
+    flags=bit.bor(NEEDARG, EXTRA, TRLBAR, NOTRLCOM, CMDWIN),
+    addr_type=ADDR_LINES,
+    func='ex_tearoff',
+  },
```

Comment:
Oh, right. It's sorted on the *short* version. I still say `:tearoff` should be removed, though I'm not going to debate it.